docs/getting-started.md

```diff
@@ -45,10 +45,10 @@ Releases are posted [on GitHub][releases]. Follow the link and download the late
 start with `ev3-`, Raspberry Pi starts with `rpi-` and Raspberry Pi 2 starts
 with `rpi2-`.
 
-Downloading the `.zip` file is recommended for
-Windows/Mac and the `.xz` file is recommended for Linux. The `.xz` is smaller,
-but it requires that you have additional software installed on Windows/Mac to
-decompress it (such as [7-Zip] on Windows).
+For most users, downloading the `.xz` file will be the best option, but the `.zip`
```

Comment:
The only case I can think of that zip files are helpful is if someone is on Windows and does not want to use Etcher; then things get complicated. But other than that, `.xz`s should be all that's needed.

The link, meanwhile, is a bit more difficult; I vaguely recall a "latest release" link, but I doubt it would be happy with the separate file names and different releases for different platforms. We'll have to edit that link in manually.